src/main/java/io/stargate/sgv2/jsonapi/service/resolver/DeleteOneCommandResolver.java

```diff
@@ -131,6 +133,23 @@ private FindCollectionOperation getFindOperation(
           false);
     }
 
+    // BM25 search / sort?
+    SortExpression bm25Expr = SortClauseUtil.resolveBM25Search(sortClause);
+    if (bm25Expr != null) {
```

Comment:
Until handling (and verification) added, important to not just quietly ignore $lexical sort, hence explicit check & fail.